conf/broker.conf

```diff
@@ -112,6 +112,17 @@ maxUnackedMessagesPerConsumer=50000
 # check and dispatcher can dispatch messages without any restriction
 maxUnackedMessagesPerSubscription=200000
 
+# Max number of unacknowledged messages allowed per broker. Once this limit reaches, broker will stop dispatching
+# messages to all shared subscription which has higher number of unack messages until subscriptions start
+# acknowledging messages back and unack count reaches to limit/2. Using a value of 0, is disabling
+# unackedMessage-limit check and broker doesn't block dispatchers
+maxUnackedMessagesPerBroker=30000000
+
+# Once broker reaches maxUnackedMessagesPerBroker limit, it blocks subscriptions which has higher unacked messages
+# than this percentage limit and subscription will not receive any new messages until that subscription acks back
+# limit/2 messages   
+unAckMsgSubscriptionPercentageLimitOnBrokerBlocked=0.16
```

Comment:
actually this percentage helps to block dispatchers when per-broker unackMsg limit reaches. 
eg: per-broker limit is = 30M so, when broker reaches this limit, it blocks all the dispatchers which has unack msg > 48K (30M * 0.16%)
Therefore, it is still percentage but as broker can have lot many dispatcher so, its value is less than 0.